dle/data/models.py

```diff
@@ -40,6 +45,12 @@ class DrugLabel(models.Model):
     link = models.URLField()
     "link is url to the external data source website"
 
+    class Meta:
+        constraints = [
+            # add a unique constraint to prevent duplicate entries
+            models.UniqueConstraint(fields=["source", "product_name", "version_date"], name="unique_dl")
```

Comment:
<img width="778" alt="image" src="https://user-images.githubusercontent.com/5684548/160703391-0f957818-e3ba-4dc0-a42b-e256adccc992.png">
FDA drug labels are unique on NDC code, not product name

Reply:
Looks like there can be NDC Codes per drug label. I'm thinking to modify to source_product_number instead of the product_name: `["source", "source_product_number", "version_date"]`

Reply:
Sounds good!